functions/patch3API.js

```diff
@@ -0,0 +1,39 @@
+const functions = require("firebase-functions");
+const cors = require("cors")({ origin: true });
+
+const admin = require("firebase-admin");
+
+const db = admin.firestore();
+
+exports.patch3 = functions.https.onRequest((request, response) =>{
+    cors(request, response, () =>
+    {
+        const OrgDomain = request.body.data.OrgDomain;
+        const oldField = request.body.data.FieldName;
+        const oldFieldValue = request.body.data.FieldValue;
+        const newField = request.body.data.NewField;
+        const newFieldValue = request.body.data.NewFieldValue;
```

Comment:
The field name for new and old field is same i believe, so we don't need two different field names.
Let have only one field name, with old value and new value.